app/integrations/discord/status_notices/unauthorized.rb

```diff
@@ -1,12 +1,21 @@
 module Discord
   module StatusNotices
     class Unauthorized
-      CONTENT = '⚠️ Could not create message: please make sure the bot has' \
-                     ' permission to send messages on this channel'.freeze
+      include Rails.application.routes.url_helpers
+
+      CONTENT = '⚠️ I couldn\'t share the message here. Please make sure' \
+                     ' I have permission to send messages on this channel.' \
+                     ' In the meantime, [click here](%<message_url>s) to see it.'.freeze
+
+      attr_reader :message
+
+      def initialize(message)
+        @message = message
+      end
 
       def build
         DiscordEngine::Message.new(
-          content: CONTENT,
+          content: format(CONTENT, message_url: message_url(message.id)),
```

Comment:
passing the whole `message` object and letting it resolve which field to use as identifier to build the path is better. That way when we work on adding a uuid to the message and using it in the path, you won't have to edit this line again :)

```suggestion
          content: format(CONTENT, message_url: message_url(message)),
```